templates/layout.html

```diff
@@ -74,8 +74,8 @@
     <div class="row justify-content-md-center">
       <div class="col-9 content-wrap">
         <div class="page py-3">
-          <a name="top"></a>
-          <h1 class="page-title pb-3 mb-4">{% block header %}{{ this.title }}{% endblock %}</h1>
+          <a></a>
+          <h2 class="pb-2"><strong>{% block header %}{{ this.title }}{% endblock %}</strong></h2>
```

Comment:
Why has the styling of the header changed? Sorry, I missed this in my previous review.